components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```
@@ -69,6 +74,153 @@ class DbContext:
+        try:
+            snapshot = self._collect_snapshot()
+        except Exception:
+            self._snapshot = None
+            logger.exception("Failed to collect archive storage metrics.")
+        else:
+            if not self._stop_event.is_set():
+                self._snapshot = snapshot
+
+    def _collect_snapshot(self) -> tuple[int, int]:
+        with (
+            closing(self._sql_adapter.create_connection(True)) as db_conn,
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)